conan.cmake

```diff
@@ -516,7 +516,7 @@ macro(conan_check)
     set(oneValueArgs VERSION)
     cmake_parse_arguments(CONAN "${options}" "${oneValueArgs}" "" ${ARGN})
 
-    find_program(CONAN_CMD conan)
```

Comment:
AFAIK it's work w/o this change. Just add $HOME/.local/bin to the user's PATH. Isn't?

Reply:
Nope, as I remember wasn't working. Because conan itself will not work without adding $HOME/.local/bin to PATH when it is installed in home folder. So this was needed fix for cmake.